src/main/java/org/folio/oaipmh/processors/OaiPmhJsonParser.java

```diff
@@ -54,4 +59,8 @@ public void end() {
       throw e;
     }
   }
+
+  public List<String> getErrors() {
+    return errors;
```

Comment:
If this list of errors is not supposed to be changed, wouldn't be better to return immutable list here?

Reply:
@obozhko-folio , I think in that case it is possible to use mutable list as well as there is no any logic to change errors list.